lib/Report/TimeDisconnectedSummary.php

```diff
@@ -340,12 +341,21 @@ public function getResults(SanitizerInterface $sanitizedParams)
 
         $displayBody = 'FROM `display` ';
 
-        if ($tags != '') {
+        if ($groupBy === 'displayGroup') {
+            $displaySelect .= ', displaygroup.displayGroup, displaygroup.displayGroupId ';
+        }
+
+        if ($tags != '' || $groupBy === 'displayGroup') {
             $displayBody .= 'INNER JOIN `lkdisplaydg`
                         ON lkdisplaydg.DisplayID = display.displayid
                      INNER JOIN `displaygroup`
-                        ON displaygroup.displaygroupId = lkdisplaydg.displaygroupId
-                         AND `displaygroup`.isDisplaySpecific = 1 ';
+                        ON displaygroup.displaygroupId = lkdisplaydg.displaygroupId ';
+
+            if ($groupBy === 'displayGroup') {
```

Comment:
If I filter by a tag which happens to be assigned to several displays, I will get a those displays as my only results. If I then group by display group without changing my tag filter, will I get other displays included in the results because of this clause?